File: Lista3/Exec1/src/exec1/PessoaFisica.java
/**
 * Classe que cria a Pessoa Física a partir da Pessoa
 */
public class PessoaFisica extends Pessoa{

    double aniv, cpf;
    String ecivil;
    /**
     * Construtor que a partir da herança da classe pessoa cria a Pessoa Física
     * @param nome
     * @param endereco
     * @param email
     * @param telefone
     * @param cpf
     * @param aniversario
     * @param estadocivil
     */
    public PessoaFisica(String nome, String endereco, String email, double telefone, double cpf, double aniversario, String estadocivil){
        super(nome, endereco, email, telefone);
        this.cpf = cpf;
        this.aniv = aniversario;
        this.ecivil = estadocivil;
    }

    // GETTERS 
    public double getCPF(){
        return this.cpf;
    }
    public double getAniv(){
        return this.aniv;
    }
    public double getECivil(){
        return this.cpf;
    }

    // SETTERS
    public void setCPF(double cpf){
        this.cpf = cpf;
    } 
    public void setAniv(double aniv){
        this.aniv = aniv;
    }
    public void setECivil(String estcvivil){
        this.ecivil = estcvivil;
    }

}

File: Lista4/exec1/pessoaFIsica.py
from pessoa import *

class pessoaFisica():

    def __init__(self, nome: str, endereco: str, email: str, tel: float, cpf: str, aniv: float, ecivil: str):
        super(self, nome, end, email, tel)
        self.cpf = cpf
        self.aniv = aniv
        self.ecivil = ecivil

    
    # getters
    def getCPF(self):
        return self.cpf
    def getAniv(self):
        return self.aniv
    def getEcivil(self):
        return self.ecivil

    # setters
    def setCPF(self, cpf):
        self.cpf = cpf
    def setAniv(self, aniv):
        self.eniv = aniv
    def setEcivil(self, ecivil):
        self.ecivil = ecivil
